pong-game/.env.example

```diff
@@ -27,8 +27,11 @@ REDIS_PASSWORD=redis_password
 
 # Monitoring
 ## ELK
+ELASTIC_PID=""
+ELASTIC_USERNAME=devops_elk
 ELASTIC_PASSWORD=elasticpassword
 KIBANA_PASSWORD=kibanapassword
+KIBANA_ENCRYPTION_KEY=kibanaencryptionkey_1234567890_need_to_be_32_chars_long
 STACK_VERSION=8.16.0
 LICENSE=basic
```

Comment:
should we put "non-sensitive" variables in the env ? we can discuss it all together tomorrow afternoon